### PR TITLE
Backport changes to Octopus

### PR DESCRIPTION
**Pull request by MrTimscampi**
**Changes**

Backports the tag and path changes for Octopus to 10.5.z

**Issues**

### CODE REVIEW

**Comment by MrTimscampi** [comment]
@JustAMan Anything that needs to be added aside from tag and paths?

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "jellyfin-noto": "https://github.com/jellyfin/jellyfin-noto",
     "jquery": "^3.4.1",
     "jstree": "^3.3.7",
-    "libass-wasm": "https://github.com/jellyfin/JavascriptSubtitlesOctopus",
+    "libass-wasm": "https://github.com/jellyfin/JavascriptSubtitlesOctopus#4.0.0-jf",
```
Comment (JustAMan) [inline package.json:34]: I wonder whether this is needed given I've tagged https://github.com/jellyfin/jellyfin-web/pull/1005 to be backported as well, won't it create conflicts?

**Comment by JustAMan** [comment]
> @JustAMan Anything that needs to be added aside from tag and paths?

I think all is fine as long as it applies cleanly along with https://github.com/jellyfin/jellyfin-web/pull/1005 being cherry-picked here

**Comment by MrTimscampi** [comment]
> > @JustAMan Anything that needs to be added aside from tag and paths?
> 
> I think all is fine as long as it applies cleanly along with #1005 being cherry-picked here

I removed the change to package.json, since it'll conflict with #1005. This should be good to go, then.

**Comment by JustAMan** [comment]
@joshuaboniface note that this PR being merged _requires_ #1005 to be backported, too, and vice versa.

**Pull request merged**